cloudvolume/cloudvolume.py

```diff
@@ -263,7 +266,7 @@ def create_new_info(cls,
       chunk_size: int (x,y,z), dimensions of each downloadable 3D image chunk in voxels
       compressed_segmentation_block_size: (x,y,z) dimensions of each compressed sub-block
         (only used when encoding is 'compressed_segmentation')
-      mip_num: (int), the number of mip levels.
+      max_mip: (int), the maximum mip level id.
```

Comment:
I typically call this parameter `num_mips`. We should be careful about introducing new parameters then changing their names... That will break other people's code (eventually). It's probably okay here because it's so new.

Reply:
I noticed that we need to use `+1` if using num_mips, I hate things like this. It is easier to introduce off by 1 bugs.

Reply:
This is a good idea. Thanks for explaining the reasoning.